addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3102,7 +3102,7 @@ export class OdooEditor extends EventTarget {
             // Move selection if next character is zero-width space
             if (nextCharacter === '\u200B') {
                 focusOffset += 1;
-                while (focusNode && !focusNode.textContent[focusOffset]) {
+                while (focusNode && !focusNode.textContent[focusOffset] || !closestElement(focusNode).isContentEditable) {
```

Comment:
Why is this needed but wasn't before ?

Reply:
I think it's a legitimate bugfix which is not related to my change, but was not creating any visible problem prior to this change.

Basically we want to avoid sanitizing elements that are `contenteditable=false`.
I can add this change in a separe commit and/or add a comment if you want.